test_scripts/Resumption/Handling_errors_from_HMI/060_resume_IVD_subscription_after_erroneous_response_2_apps_waiting_for_IVD_response_for_app2.lua

```diff
@@ -0,0 +1,108 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description:
+-- In case:
+-- 1. Subscriptions for moduleType_1 and moduleType_2 are added by app1
+-- 2. Subscriptions for moduleType_2 and moduleType_3 are added by app2
+-- 3. Unexpected disconnect and reconnect are performed
+-- 4. App1 and app2 re-register with actual HashId
+-- 5. RC.GetInteriorVehicleData(moduleType_1, subscribe = true) and
+--  RC.GetInteriorVehicleData(moduleType_2, subscribe = true) related to app1
+--  are sent from SDL to HMI during resumption
+-- 6. RC.GetInteriorVehicleData(moduleType_3, subscribe = true) related to app2
+--  is sent from SDL to HMI during resumption
+-- 7. HMI responds with successful resultCode to RC.GetInteriorVehicleData(moduleType_1) request related to app1
+--  HMI responds with errorneous resultCode to RC.GetInteriorVehicleData(moduleType_2) request related to app1
+--  HMI responds with successful resultCode to RC.GetInteriorVehicleData(moduleType_3) request related to app2
+-- SDL does:
+-- 1. send revert RC.GetInteriorVehicleData(moduleType_1, subscribe = false) related to app1 to HMI
+-- 2. send RC.GetInteriorVehicleData(moduleType_2, subscribe = true) related to app2 to HMI
+-- 8. HMI responds with successful resultCode to RC.GetInteriorVehicleData(moduleType_2) request related to app1
```

Comment:
3. HMI responds with successful resultCode to RC.GetInteriorVehicleData(moduleType_1) request related to app1